packages/browserslist-config/index.js

```diff
@@ -0,0 +1 @@
+module.exports = [ '> 1%', 'iOS >= 10', 'last 2 versions', 'not ie <= 10', 'Safari >= 10' ];
```

Comment:
Hmm, this was more readable as separate lines. I wish I could stop Prettier doing this.

Reply:
Prettier doesn't have any formatting "rules" that would be distinguishable from each other. The comment just skips formatting for the next AST node.

Reply:
Thanks @jsnajdr . I added the comment (because I prefer readability over Prettier), lemme know if you'd prefer not to have it.